tests/conftest.py

```diff
@@ -458,7 +457,7 @@ def minicircle_grain_clear_border(minicircle_grain_mask: np.array) -> Grains:
 
 @pytest.fixture
 def minicircle_grain_remove_noise(minicircle_grain_clear_border: np.array) -> Grains:
-    """Cleared borders."""
+    """Remove Noise."""
```

Comment:
I thought the agreed term was "crap" :wink: 

```suggestion
    """Test Removing Noise."""
```